cli/popper/commands/cmd_archive.py

```diff
@@ -115,27 +116,48 @@ def upload_snapshot(service_url, params, filename):
             "{} and the file id is {}.".format(deposition_id, file_id)
         )
     else:
-        pu.fail(
-            "Status {}: Failed to upload your snapshot. Please "
-            "try again.".format(r.status_code)
-        )
+        pu.fail("Failed to upload your snapshot. Please try again.")
 
     return deposition_id
 
 
 def add_metadata(metadata_url, params):
     """Receives the metadata_url and other parameters, reads the metadata and
     adds that to the deposit. Returns the publish_url."""
-    data = {
-        'metadata': {
-            'title': "This is the title",
-            'upload_type': 'software',
-            'description': "A suitable description",
-            'creators': [
-                {'name': 'Doe, John', 'affiliation': 'popper'},
-            ]
-        }
-    }
+
+    # Get the metadata from the .popper.yml file
+    data = pu.read_config()['metadata']
+
+    try:
+        # Change abstract to description
+        data['description'] = '<p>' + data['abstract'] + '</p>'
+        del data['abstract']
+
+        # Collect the authors
+        creators = []
+        for key in list(data.keys()):
```

Comment:
would it be possible to sort the list? We might have `author1`, `author2`, etc., and user might want to keep that order when archiving.